### DISCUSSION TITLE
获取消息时出现错误: get value error: leveldb: not found

### DISCUSSION THREAD

**Discussion by Rootive** [Mrs4s/go-cqhttp#2218] [Q&A] [answered]
相关配置：
![image](https://github.com/Mrs4s/go-cqhttp/assets/81627800/5bdcfa5f-3b1a-4c5d-84b3-e16a12ac2d8f)
报错内容：
![image](https://github.com/Mrs4s/go-cqhttp/assets/81627800/a40199aa-46d3-4054-aa64-a5717c4300d7)
![image](https://github.com/Mrs4s/go-cqhttp/assets/81627800/52ba002c-766b-431f-abf3-07baebca373a)
测试代码：
![image](https://github.com/Mrs4s/go-cqhttp/assets/81627800/ea8e5aa6-658a-457c-a41e-bbb9d627b27c)
我从日志中，从以下角度摘取message_id：
1.send_msg返回的message_id
2.message_sent上报的message_id（与1一致，符合预期）
3.直接从上报的message摘取message_id
都会返回上述错误。

请问有uu解决过类似的问题，或者经验吗？

**Comment by xiangxiangxiong9** [comment, marked as answer]
尝试把 
```go
get := structs.GetMessage{
    MessageID: -912357563,
}
```
中的 `MessageID` 换为 `message_id`

**Reply by Rootive** [reply]
谢谢，是我犯二了。

**Reply by Rootive** [reply]
你好。私聊时，用通过解析reply的CQ码获得message_id，来get_msg仍然会报leveldb: not found。

**Reply by xiangxiangxiong9** [reply]
无法尝试复现 [请确认是否写错了 message_id]

**Reply by Rootive** [reply]
我尝试复现了，已经提交issue：https://github.com/Mrs4s/go-cqhttp/issues/2226
